app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -83,32 +88,12 @@ protected void onCreate(Bundle savedInstanceState) {
     bnv = findViewById(R.id.bottom_navigation);
     splash = findViewById(R.id.relative_layout_splash);
     bnv.setVisibility(View.GONE);
+    areaButtonsPressed = new HashSet<>();
+    paymentButtonsPressed = new HashSet<>();
     if (getSupportActionBar() != null) {
       getSupportActionBar().hide();
     }
-
-    ConnectionUtilities con = new ConnectionUtilities(this);
-    if (!con.isNetworkAvailable()) {
-      cafeList = new ArrayList<>();
-      if (JsonUtilities.parseJson(dbHelper.getLastRow(), getApplicationContext()) != null) {
-        cafeList = JsonUtilities.parseJson(dbHelper.getLastRow(), getApplicationContext());
-      }
-      Collections.sort(cafeList);
-      currentList = cafeList;
-      searchList = cafeList;
-
-      mRecyclerView.setHasFixedSize(true);
-      LinearLayoutManager layoutManager =
-          new LinearLayoutManager(getApplicationContext(), LinearLayout.VERTICAL, false);
-      mRecyclerView.setLayoutManager(layoutManager);
-
-      listAdapter =
-          new MainListAdapter(
-              getApplicationContext(), MainActivity.this, cafeList.size(), cafeList);
-      mRecyclerView.setAdapter(listAdapter);
-    } else {
-      new ProcessJson().execute("");
-    }
+    new ProcessJson().execute("");
```

Comment:
@jyc979 can we just make it so we dont need to pass in any parameters instead of passing in empty string

Reply:
just saw this will keep this in mind for next update